docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -56,54 +56,54 @@ flavors.
-|standard.small  |2|1.9  |80 |0 |80 |0.9  |![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|0.5  |
-|standard.medium |3|3.9 |80 |0 |80 |1.3|![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|1    |
-|standard.large  |4|7.8 |80 |0 |80 |1.9|![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|2    |
-|standard.xlarge |6|15 |80 |0 |80 |2.5|![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|4    |
-|standard.xxlarge|8|31 |80 |0 |80 |3.8|![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|8    |
-|standard.3xlarge|8|62 |80 |0 |80 |7.7|![](/img/circle_icons/p100.svg)![](/img/circle_icons/r100.svg)![](/img/circle_icons/n100.svg)|16   |
+|standard.tiny   |1|0.9 |80 |0 |80 |0.9  |![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|0.25 |
+|standard.small  |2|1.9  |80 |0 |80 |0.9  |![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|0.5  |
+|standard.medium |3|3.9 |80 |0 |80 |1.3|![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|1    |
+|standard.large  |4|7.8 |80 |0 |80 |1.9|![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|2    |
+|standard.xlarge |6|15 |80 |0 |80 |2.5|![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|4    |
+|standard.xxlarge|8|31 |80 |0 |80 |3.8|![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|8    |
+|standard.3xlarge|8|62 |80 |0 |80 |7.7|![Icon for power redundancy level FULL](../../img/circle_icons/p100.svg)![Icon for root disk data redundancy level FULL](../../img/circle_icons/e100.svg)![Icon for network reachability redundancy level FULL](../../img/circle_icons/e100.svg)|16   |
```

Comment:
Most of these icons seem wrong, too many `E` 

![Screenshot from 2024-03-13 10-01-30](https://github.com/CSCfi/csc-user-guide/assets/2276349/e88f5b49-4d12-4894-a256-84ebf64094f3)